.github/workflows/pdf-visual-qa.yml

```
@@ -0,0 +1,198 @@
+            echo ""
+            echo "📄 Validating: $pdf"
+            echo "================================"
+            
+            python -m src.kindlemint.validators.integrated_pdf_validator "$pdf" || true
+            
+            # Check if visual report was generated
+            report_name="${pdf%.pdf}_visual_qa_report.json"
+            if [ -f "$report_name" ]; then
+              echo "📊 Visual QA Report Summary:"
+              python -c "
+import json
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 85: could not find expected ':'

(syntax-check)